addons/website/views/snippets/s_dynamic_snippet_carousel.xml

```diff
@@ -9,7 +9,6 @@
         <t t-call="website.s_dynamic_snippet_options_template">
             <t t-set="snippet_name" t-value="snippet_name"/>
             <t t-set="snippet_selector" t-value="snippet_selector"/>
-            <t t-set="grouping_message">Items per slide</t>
```

Comment:
Note that this line still survived in `s_dynamic_snippet.xml` and `s_events.xml`. I guess it should also be removed there.